bom/pom.xml

```diff
@@ -38,12 +38,14 @@
     <influxdb.version>1.2.4-alpine</influxdb.version>
     <jackson.version>2.9.0</jackson.version>
     <java-base-image.name>openjdk:8u171-jre-slim</java-base-image.name>
-    <jjwt.version>0.7.0</jjwt.version>
+    <jaxb.api.version>2.3.0</jaxb.api.version>
+    <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
+    <jjwt.version>0.9.0</jjwt.version>
```

Comment:
doesn't it work with 0.7.0?

Reply:
@ctron have you tried using 0.7.0?

Reply:
I re-test with 0.7.0 and it does work. So I did change the version back to 0.7.0. However it still requires the extra dependency on jaxb.